apps/algorithm/recommend/src/recommender.py

```diff
@@ -4,6 +4,13 @@
 import requests
 import os
 
+RECOMMENDER_VERSION = "1.0.1"
+RECOMMENDER_URL = f"https://github.com/UVicMartletplace/martletplace/releases/download/recommender-v{RECOMMENDER_VERSION}/"
+PROCESSED_DATA_URL = "processed_data.csv"
+COSINE_URL = "cosine_similarity_matrix.npy"
+ITEM_VECTORS_URL = "normalized_item_vectors.npy"
+TRAINING_DIR_URL = "/app/src/training/"
+
```

Comment:
Perhaps these should be environment variables but this was supposed to be a temporary solution anyway so it doesn't matter.